### PR TITLE
Active media url fix

### PR DESCRIPTION
**Pull request by jmourell**
End campaign saga early when there is no media in a new campaign.
Remove default data

### CODE REVIEW

File: src/sagas/CampaignSaga.js [hunk 1]
```diff
@@ -21,7 +21,9 @@ export function* campaignSaga({ type, data }) {
       }
 
     }
+    console.log("Campaign API response:" ,campaignApiResponse);
```
Comment (wilandrade) [inline src/sagas/CampaignSaga.js]: Please remove console.log

File: src/components/dashboardCards/ExperiencesTable.jsx [hunk 2]
```diff
@@ -80,14 +80,7 @@ class ExperiencesTable extends Component {
...
                     ))
-                  ) : (
-                    <tr>
-                      <td>Garrett Winters</td>
-                      <td>Video</td>
-                      <td>2013/07/01</td>
-                      <td>63</td>
-                    </tr>
-                  )}
+                  ) : null}
```
Comment (wilandrade) [inline src/components/dashboardCards/ExperiencesTable.jsx:83]: What do we end up rendering where there is no media like this? Does it show up as null on the website, or is it just empty?

File: src/components/Dashboard.jsx [hunk 3]
```diff
@@ -25,18 +25,6 @@ class Dashboard extends Component {
     }
     this.props.fetchCampaignData(this.props.admin);
   }
-  getMediaUrl() {
```
Comment (wilandrade) [inline src/components/Dashboard.jsx:28]: Why are we removing this?

**Review by wilandrade — CHANGES_REQUESTED** [review]

File: src/components/Dashboard.jsx (hunk 3 above)
Comment (jmourell) [reply src/components/Dashboard.jsx:28]: It is no longer passed as props in this card.

File: src/components/dashboardCards/ExperiencesTable.jsx (hunk 2 above)
Comment (jmourell) [reply src/components/dashboardCards/ExperiencesTable.jsx:83]: It shows up as an empty table with just titles.

**Pull request merged**